static/app/views/insights/browser/webVitals/utils/useRunSeerAnalysis.tsx

```diff
@@ -45,7 +50,8 @@ export function useRunSeerAnalysis({
           vital: webVital,
           score: projectScore[`${webVital}Score`],
           transaction,
-          traceId: webVitalTraceSamples[webVital],
+          traceId: webVitalTraceSamples[webVital]?.trace,
+          timestamp: webVitalTraceSamples[webVital]?.timestamp,
```

Comment:
I'm curious, in your code I don't see how this could go wrong. But it does feel like weird to me pass in the timestamp when we can lookup it up from the trace id in the backend.

I guess this way is more performant and less annoying?

Reply:
I was thinking there may be scenarios where you want to explicitly set the timestamp for an issue event without a trace available. But also this seemed to easiest way.